kitsune/dashboards/tests/test_readouts.py

```diff
@@ -606,11 +594,9 @@ def test_untranslated(self):
     def test_by_product(self):
         """Test the product filtering of the readout."""
         p = product(title='Firefox', slug='firefox', save=True)
-        d = document(title='Template:test', save=True)
-        revision(is_approved=True,
-                 is_ready_for_localization=True,
-                 document=d,
-                 save=True)
+        d = TemplateDocumentFactory()
+        d = document(title='Template:test', category=TEMPLATES_CATEGORY, save=True)
```

Comment:
The previous line assigns a value to d, then this line assigns another value to d.

I mention that in case the first line is superfluous. If it is, you can ditch the line. If it's not, then ditch the `d =`, but probably worth adding a comment about why you're creating two since the test seems to only be about one.